wbooks/src/app/screens/BookDetail/styles.tsx

```diff
@@ -6,7 +6,7 @@ const styles = StyleSheet.create({
     flex: 1,
     backgroundColor: Colors.secondary,
     paddingHorizontal: 20,
-    paddingTop: 10
+    paddingTop: 70
```

Comment:
Why this big padding?

Reply:
I needed it so that the list is not hidden behind the header background. There's no need? is there a better way?
![image](https://user-images.githubusercontent.com/77977855/108645081-51b71e00-7490-11eb-8112-039696b01aea.png)

Reply:
Ok